pkg/scheduler/api/test_utils.go

```diff
@@ -72,6 +74,7 @@ func buildPod(ns, n, nn string, p v1.PodPhase, req v1.ResourceList, owner []meta
 	return &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			UID:             types.UID(fmt.Sprintf("%v-%v", ns, n)),
+			Annotations:     map[string]string{v1alpha1.GroupNameAnnotationKey: "testpodgroup"},
```

Comment:
Remove this change.

Reply:
removing this is causing some unit tests to fail. I'll look into this in a bit.

Reply:
Oh, Sorry. I thought it was added by us. Let's leave it if it is not added by us.